readme.md

```diff
@@ -33,6 +33,28 @@ client.getPosts(function( error, posts ) {
 
 More usage examples can be found in the `examples` directory.
 
+## Basic Authentication
```

Comment:
This does not belong in its own section and it's missing from the section it does belong in. Please document the option in the constructor.